r-package/grf/R/plot.R

```diff
@@ -80,6 +80,10 @@ export_graphviz <- function(tree) {
 }
 
 #' Plot a GRF tree object.
+#' 
+#' If you would like to save the plots, you can do so by installing the DiagrammeRsvg package and running the command: 
```

Comment:
Maybe add this to the @examples section as an example of how to save the plot from the command line?